codec-http2/src/main/java/io/netty/handler/codec/http2/AbstractHttp2StreamChannel.java

```diff
@@ -97,12 +96,12 @@ public Handle newHandle() {
     private static final AtomicIntegerFieldUpdater<AbstractHttp2StreamChannel> UNWRITABLE_UPDATER =
             AtomicIntegerFieldUpdater.newUpdater(AbstractHttp2StreamChannel.class, "unwritable");
 
-    private static void windowUpdateFrameWriteComplete(ChannelFuture future, Channel streamChannel) {
+    private static void windowUpdateFrameWriteComplete(Channel streamChannel, Future<?> future) {
```

Comment:
nit: Why this change ?

Reply:
It's used as a method reference, and therefor the signature needs to be compatible with the `FutureContextListener` interface.